control-plane-operator/controllers/hostedcontrolplane/v2/storage/component.go

```diff
@@ -47,7 +47,6 @@ func NewComponent() component.ControlPlaneComponent {
 		WithManifestAdapter(
 			"azure-disk-csi-config.yaml",
 			component.WithAdaptFunction(adaptAzureCSIDiskSecret),
-			component.WithPredicate(isAroHCP),
```

Comment:
those manifests will now be created for all platforms. Should only be enabled for Azure
```
component.EnableForPlatform(hyperv1.AzurePlatform)
```